packages/manager/apps/dedicated/client/app/billing/autoRenew/agreements/details/user-agreements-details.controller.js

```diff
@@ -31,8 +32,8 @@ angular
         this.alreadyAccepted = false;
 
         $q.all([
-          Service.getAgreement($stateParams.id),
-          Service.getContract($stateParams.id),
+          Service.getAgreement(toInteger($stateParams.id)),
```

Comment:
Not possible to specify with ui-router declaration than the param is an int ? 
Also could state params be injected through a resolve instead ? To avoid its use in the controller

Reply:
The id is now resolved in the resolver route. I don't think the param can be declared as an int directly